src/MagnumPlugins/BasisImageConverter/BasisImageConverter.h

```diff
@@ -0,0 +1,111 @@
+#ifndef Magnum_Trade_BasisImageConverter_h
+#define Magnum_Trade_BasisImageConverter_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2019 Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Class @ref Magnum::Trade::BasisImageConverter
+ */
+
+#include <Magnum/Trade/AbstractImageConverter.h>
+
+#include "MagnumPlugins/BasisImageConverter/configure.h"
+
+#ifndef DOXYGEN_GENERATING_OUTPUT
+#ifndef MAGNUM_BASISIMAGECONVERTER_BUILD_STATIC
+    #ifdef BasisImageConverter_EXPORTS
+        #define MAGNUM_BASISIMAGECONVERTER_EXPORT CORRADE_VISIBILITY_EXPORT
+    #else
+        #define MAGNUM_BASISIMAGECONVERTER_EXPORT CORRADE_VISIBILITY_IMPORT
+    #endif
+#else
+    #define MAGNUM_BASISIMAGECONVERTER_EXPORT CORRADE_VISIBILITY_STATIC
+#endif
+#define MAGNUM_BASISIMAGECONVERTER_LOCAL CORRADE_VISIBILITY_LOCAL
+#else
+#define MAGNUM_BASISIMAGECONVERTER_EXPORT
+#define MAGNUM_BASISIMAGECONVERTER_LOCAL
+#endif
+
+namespace Magnum { namespace Trade {
+
+/**
+@brief BASIS image converter plugin
+
+Creates Portable Network Graphics (`*.png`) files from images with format
+@ref PixelFormat::R8Unorm / @ref PixelFormat::R16Unorm,
+@ref PixelFormat::RG8Unorm / @ref PixelFormat::RG16Unorm,
+@ref PixelFormat::RGB8Unorm / @ref PixelFormat::RGB16Unorm or
+@ref PixelFormat::RGBA8Unorm / @ref PixelFormat::RGBA16Unorm.
```

Comment:
I spent quite some time looking into how Basis handles 16-bit depth.... just to notice this is just a copypaste error.

`libBASIS` ... what, did you read this at least once? :joy:

Reply:
Oh no, I must have forgotten about this entirely documentation Block entirely, it even sill says that it creates PNG files